foundationdb/datadog_checks/foundationdb/check.py

```diff
@@ -23,8 +23,12 @@ def construct_database(self):
             return self._db
 
         # TLS options. Each option has a different function name, so we cannot be smart with it without ugly code
+        if 'tls_ca_file' in self.instance:
+            fdb.options.set_tls_ca_path(self.instance.get('tls_ca_file'))
         if 'tls_certificate_file' in self.instance:
             fdb.options.set_tls_cert_path(self.instance.get('tls_certificate_file'))
+        if 'tls_password' in self.instance:
```

Comment:
See https://apple.github.io/foundationdb/api-python.html
> `fdb.options.set_tls_password(password)`
Sets the passphrase for encrypted private key. Password should be set before setting the key for the password to be used.